src/scripts/antithesis.zig

```diff
@@ -159,6 +180,68 @@ fn push_image(shell: *Shell, image: Image, tag: []const u8) !void {
     );
 }
 
+fn trigger_test(
+    shell: *Shell,
+    tag: []const u8,
+    antithesis_user: []const u8,
+    antithesis_password: []const u8,
+) !void {
+    const commit = try shell.exec_stdout("git rev-parse HEAD", .{});
+
+    var body = std.ArrayList(u8).init(shell.gpa);
+    defer body.deinit();
+
+    try std.json.stringify(.{
+        .params = .{
+            .@"custom.duration" = "0.5",
+            .@"antithesis.images" = try shell.fmt("workload:{s};replica:{s}", .{ tag, tag }),
+            .@"antithesis.config_image" = try shell.fmt("config:{s}", .{tag}),
+            .@"antithesis.description" = commit,
+            .@"antithesis.report.recipients" = "oskar@tigerbeetle.com",
+        },
+    }, .{}, body.writer());
+
+    var client = std.http.Client{ .allocator = shell.gpa };
+    defer client.deinit();
+
+    const user_pass = try shell.fmt("{s}:{s}", .{ antithesis_user, antithesis_password });
+    const b64 = std.base64.url_safe.Encoder;
+
+    const auth = try shell.gpa.alloc(u8, b64.calcSize(user_pass.len));
```

Comment:
I'd rather stack-allocate a kilobyte and use that. If someone tries to use War and Peace as their password, we should fail _before_ we get to doing an HTTP request

Reply:
Actually, the same goes for the json above I think? Would be good to impose static limits on the amount of data we send over the wire